src/Data/Map/Internal.purs

```diff
@@ -90,11 +93,8 @@ instance ordMap :: (Ord k, Ord v) => Ord (Map k v) where
 instance showMap :: (Show k, Show v) => Show (Map k v) where
   show m = "(fromFoldable " <> show (toAscArray m) <> ")"
 
-instance semigroupMap :: Ord k => Semigroup (Map k v) where
-  append = union
-
-instance monoidMap :: Ord k => Monoid (Map k v) where
-  mempty = empty
+instance altMap :: Ord k => Alt (Map k) where
+  alt = union
```

Comment:
Could you also add a Plus instance (and derive it for the unbiased newtype too)?

Reply:
Fixed.